cse-runner-app/src/main/java/com/farao_community/farao/cse/runner/app/models/ForcedPras.java

```diff
@@ -0,0 +1,25 @@
+package com.farao_community.farao.cse.runner.app.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class ForcedPras {
```

Comment:
This should be put in the API and called something like ExternalProcessParameters to make it more generic. As initialDichtomyStep is part of CseRequest already, I think it would be better to handle it in the CSE adapter.